doc/PACKAGERS.md

```diff
@@ -38,50 +38,54 @@ change how Rizin is built.
 First, we suggest you use the options `-Dpackager_version=<package-version>`
 and `-Dpackager=<packager>` to help us track the version of Rizin users are
 using, because these options are used when you run `rizin -v`. In this way
-when a user reports a issue and he provides his `rizin -v` output, we know
-how Rizin was built. Below you can see an example of how rizin uses the
+when a user reports an issue and provides his `rizin -v` output, we know
+how Rizin was built. Below, you can see an example of how rizin uses the
 additional information:
 ```
 rizin 0.2.0-git @ linux-x86-64, package: 0.2.0-git (rizinorg)
 commit: 84d2892e7210dc3ced88ae006ba5a9502f4847c8, build: 2021-01-29__09:35:03
 ```
 
 Then, to define the base install location for Rizin use the `--prefix` flag when
-invoking `meson`. For system installs it is common to use `/usr`. If in doubt
+invoking `meson`. For system installs it is common to use `/usr`. If in doubt,
 check your distributions packaging guidelines.
 
 If you do not use `/usr` as a prefix, you may want to use `-Dlocal=disabled` to
 avoid `RPATH` in the installed binaries.
 
 If you want to specify different directories for binaries, libraries, header
-files, etc. you may want to look at `--bindir`, `--libdir`, `--includedir` or
+files, etc., you may want to look at `--bindir`, `--libdir`, `--includedir` or
 check `meson setup --help` for more options. For extra control over the
 directories used by Rizin, have a look at options `rizin_sdb`, `rizin_zigns`,
```

Comment:
remove rizin_zigns

Reply:
also modify rz-sign to replace `.Nd rizin zignature utility` with `.Nd rizin signature utility`